### DISCUSSION TITLE
Read the table filter in other components or widgets

### DISCUSSION THREAD

**Discussion by mshanak** [filamentphp/filament#4279] [Help]
Hello,

Is there a way to read the table filters (or the filter query) in other widgets on the same page?

My issue is:
In a ListPage, I have a Widget that shows some statistics about the data displayed in the table. I need to update the statistics every time the table filter changes. 

Thank you.

**Comment by danharrin** [comment]
This will be possible in Livewire 3 (Filament 3)
